qiskit_dynamics/signals/signals.py

```diff
@@ -303,38 +290,27 @@ def __init__(
         """
         self._dt = dt
 
-        samples = Array(samples)
+        samples = unp.asarray(samples)
 
         if len(samples) == 0:
-            zero_pad = np.array([0])
+            zero_pad = unp.asarray([0])
```

Comment:
similar to the other comment - I think here it still makes sense to keep it as `np`, since this will just get dispatched to numpy.